docs/en/08_troubleshooting.md

```diff
@@ -9,38 +15,37 @@ work immediately, without being persisted or executed via cron. If this works, n
 cronjob is configured and executing correctly.
 
 If defining your own job classes, be aware that when the job is started on the queue, the job class
-is constructed _without_ parameters being passed; this means if you accept constructor args, you
-_must_ detect whether they're present or not before using them. See [this issue](https://github.com/symbiote/silverstripe-queuedjobs/issues/35)
+is constructed *without* parameters being passed; this means if you accept constructor args, you
+*must* detect whether they're present or not before using them. See [this issue](https://github.com/symbiote/silverstripe-queuedjobs/issues/35)
 and [this wiki page](https://github.com/symbiote/silverstripe-queuedjobs/wiki/Defining-queued-jobs) for
 more information.
 
-If defining your own jobs, please ensure you follow PSR conventions, i.e. use "YourVendor" rather than "SilverStripe".
+If defining your own jobs, please ensure you follow PSR conventions, i.e. use "YourVendor" rather than "Silverstripe".
```

Comment:
```suggestion
If defining your own jobs, please ensure you follow PSR conventions, i.e. use `YourVendor` rather than `SilverStripe`.
```
This is talking about namespaces (at least I'm 90% sure) so the double capital is correct.

Reply:
Updated